monitoring/poison_message_queue_finder.py

```diff
@@ -141,5 +141,25 @@ def get_connection_stats():
             print(json.dumps(json_to_dump))
 
 
+def get_bad_message_counts():
+    response = requests.get(f'{Config.EXCEPTIONMANAGER_URL}/badmessages/summary?minimumSeenCount=4')
+    response.raise_for_status()
```

Comment:
Maybe better to log out the error and return but not raise it, it'll kill the pod and briefly take down our rabbit monitoring too if it network errors. On the other hand, this way we'll see it much more obviously if our monitor starts getting errors.